flytectl/cmd/delete/matchable_cluster_resource_attribute_test.go

```diff
@@ -130,8 +130,8 @@ func TestDeleteClusterResourceAttributes(t *testing.T) {
 		err := deleteClusterResourceAttributes(s.Ctx, []string{}, s.CmdCtx)
 		assert.NotNil(t, err)
 		assert.Equal(t,
-			fmt.Errorf("error unmarshaling JSON: while decoding JSON: json: unknown field \"InvalidDomain\""),
-			err)
+			"error unmarshaling JSON: while decoding JSON: json: unknown field \"InvalidDomain\"",
+			err.Error())
 		s.DeleterExt.AssertNotCalled(t, "DeleteProjectDomainAttributes",
```

Comment:
Due to a dependency upgrade the type of the error changed:

```console
            	Error:      	Not equal: 
            	            	expected: *errors.errorString(&errors.errorString{s:"error unmarshaling JSON: while decoding JSON: json: unknown field \"InvalidDomain\""})
            	            	actual  : *fmt.wrapError(&fmt.wrapError{msg:"error unmarshaling JSON: while decoding JSON: json: unknown field \"InvalidDomain\"", err:(*errors.errorString)(0xc0001314f0)})
            	Test:       	TestDeleteTaskResourceAttributes/attribute_deletion_invalid_file
```

Let's only compare the error message, not the type.